README.md

```diff
@@ -278,12 +287,12 @@ test resolver how to resolve abilities from the file structure.
 
 An ability unit test will be created each time you generate a new ability via
 `ember g ability <name>`. The package currently supports generating QUnit and
-Mocha style tests.  
+Mocha style tests.
 
 ### Unit testing in your app
 
 To unit test modules that use the `can` helper, you'll need to explicitly add `needs` for the
-ability and helper file like this:  
+ability and helper file like this:
```

Comment:
Looks like your editor did this and it broke the spacing here - you should be able to just place the "```" on its own line to get the same behavior.